awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -214,108 +234,73 @@ function JobTemplateForm({
           tooltip={i18n._(t`Select the inventory containing the hosts
             you want this job to manage.`)}
         >
-          <Field name="inventory">
-            {({ form }) => (
-              <>
-                <InventoryLookup
-                  value={inventory}
-                  onBlur={() => {
-                    form.setFieldTouched('inventory');
-                  }}
-                  onChange={value => {
-                    form.setValues({
-                      ...form.values,
-                      inventory: value.id,
-                      organizationId: value.organization,
-                    });
-                    setInventory(value);
-                  }}
-                  required
-                  touched={form.touched.inventory}
-                  error={form.errors.inventory}
-                />
-                {(form.touched.inventory ||
-                  form.touched.ask_inventory_on_launch) &&
-                  form.errors.inventory && (
-                    <div
-                      className="pf-c-form__helper-text pf-m-error"
-                      aria-live="polite"
-                    >
-                      {form.errors.inventory}
-                    </div>
-                  )}
-              </>
+          <InventoryLookup
+            value={inventory}
+            onBlur={() => inventoryHelpers.setTouched()}
+            onChange={value => {
+              inventoryHelpers.setValue(value.id);
+              setFormikFieldValue('organizationId', value.organization);
+              setInventory(value);
+            }}
+            required
+            touched={inventoryMeta.touched}
+            error={inventoryMeta.error}
+          />
+          {(inventoryMeta.touched || formikValues.ask_inventory_on_launch) &&
```

Comment:
while this isn't a change you made in this pr, I'm wondering if we can utilize the error system other things use, rather than throwing a custom div here.

(this is also the one place where my plan to excise formikContext might not be feasible...we might not be able to grab just from the useField calls and need the formikContext...not sure off the top of my head